.github/workflows/cd-api.yml

```
@@ -33,7 +31,7 @@ jobs:
     strategy:
       max-parallel: 1
       matrix:
-        envs: ${{ github.event_name == 'release' && fromJSON('["prod"]') || github.ref_name == 'main' && fromJSON('["dev", "staging"]') || fromJSON('["dev"]') }}
+        envs: ${{ fromJSON('["dev", "staging"]') }}
```

swap this line to just 
```suggestion
        envs: [ "dev" ]
```

I was testing the multi app deploy workflow in my example